code/game/objects/items/toys.dm

```diff
@@ -1143,6 +1143,12 @@
 	icon_state = "ipcplushie"
 	phrase = "Bwoop!"
 
+/obj/item/toy/plushie/domadice
+	name = "Domadice  plushie"
+	desc = "A marketable plushie of the Kessvalankan Domadice"
```

Comment:
this is just a suggestion, not a requirement, but i think it'd be nice if you had an extended lore description that explained what Kessvalankan is, and how Domadice relates to the GD, so that players can learn more about it via looking at the plushie